boards/arm/rpi_pico/doc/index.rst

```diff
@@ -85,13 +94,22 @@ hardware features:
      - :kconfig:option:`CONFIG_SERIAL`
      - :dtcompatible:`raspberrypi,pico-uart-pio`
 
+The rpi_pico_w board configuration will additionally support WiFi and
```

Comment:
I don't think those lines are necessary as the wifi module is not listed in the supported features